Files/Interacts/Interaction.cs

```diff
@@ -894,13 +830,17 @@ private async void Manager_DataRequested(DataTransferManager sender, DataRequest
                 }
                 else if (item.PrimaryItemAttribute == StorageItemTypes.Folder)
                 {
-                    await AssociatedInstance.FilesystemViewModel.GetFolderFromPathAsync(item.ItemPath)
-                        .OnSuccess(folderAsItem => items.Add(folderAsItem));
+                    if (await StorageItemHelpers.ToStorageItem<StorageFolder>(item.ItemPath) is StorageFolder folder)
```

Comment:
I'd also pass `AssociatedInstance` to the method like this:
```cs
await StorageItemHelpers.ToStorageItem<StorageFolder>(item.ItemPath, AssociatedInstance)
```
Difference being that this should also work inside e.g phone storage. Same thing below.

Might still be posssible to get rid of the AssociatedInstance by changing ToStorageItem() somehow..